### PR TITLE
Add DamageAlteration to Divine Castigation

### PR DESCRIPTION
**Pull request by reyzor1991**

### CODE REVIEW

**Comment by stwlam** [comment]
The feat doesn't unconditionally change the damage type to spirit.

**Comment by reyzor1991** [comment]
https://2e.aonprd.com/Feats.aspx?ID=4643&Redirected=1

``` When you cast a harm or heal spell, you can add your holy or unholy trait to it.```

``` The spell deals spirit damage when used this way```

divine-castigation is predicate from toggle

**Comment by stwlam** [comment]
Missed that there was already a toggle, sorry!

File: packs/feats/divine-castigation.json [hunk 1]
```diff
@@ -69,6 +69,30 @@
...
+                    {
+                        "or": [
+                            "item:slug:harm",
+                            "item:slug:heal"
+                        ]
+                    }
+                ],
+                "selectors": [
+                    "spell-damage"
+                ],
```
Comment (stwlam) [inline packs/feats/divine-castigation.json:93]:
```suggestion
```

Comment (reyzor1991) [reply packs/feats/divine-castigation.json:93]: Sorry, don't understand logic of delete it

After it, json would be invalid

Comment (reyzor1991) [reply packs/feats/divine-castigation.json:93]: image

Comment (stwlam) [reply packs/feats/divine-castigation.json:93]: Oh, right, this is a damage alteration rather than an item one.

**Pull request merged**